src/cc_catalog_airflow/dags/provider_api_scripts/common/requester.py

```diff
@@ -57,3 +58,38 @@ def _delay_processing(self):
         if wait >= 0:
             logging.debug(f'Waiting {wait} second(s)')
             time.sleep(wait)
+
+    def get_response_json(self, endpoint, retries=0, query_params=None,
+                          **kwargs):
```

Comment:
Please use this argument style:

```suggestion
    def get_response_json(
            self,
            endpoint,
            retries=0,
            query_params=None,
            **kwargs
    ):
```
It will make for smaller diffs in the future (we'll change the signature of this function in a future issue).